log/development.log

```diff
@@ -0,0 +1,44 @@
+  [1m[35m (0.3ms)[0m  [1m[35mCREATE DATABASE "myapp_development" ENCODING = 'unicode'[0m
```

Comment:
We also should add to the `.gitignore` the log folder, do the same that you did for `test`